src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  selectedTodo: Todo | null;
+  onShowTodo: (todo: Todo | null) => () => void;
```

Comment:
The return type of 'onShowTodo' should be a function that returns void, not a double arrow function. Replace '() => void' with '(todo: Todo | null) => void'.